CHANGES.rst

```diff
@@ -76,8 +76,9 @@ Minor changes
   is now always visible when scrolling the table. :pr:`1102` by :user:`Jérôme
   Dockès <jeromedockes>`.
 
-* Added a `DropColumnIfNull` transformer that drops columns that contain only null
-  values. :pr:`1115` by :user: `Riccardo Cappuzzo <riccardocappuzzo>`
+* Added a `DropColumnIfNull` transformer that drops columns based on how many
```

Comment:
actually DropColumnIfNull has not been added to skrub's public API (and I'm not sure if it should before we decide what to do with selectors) so here we should document the added parameter to TableVectorizer rather than DroopColumnIfNull